modules/ROOT/pages/depl-examples/ubuntu-compose/ubuntu-compose-prod.adoc

```diff
@@ -0,0 +1,384 @@
+Add a valid response eMail address for Letsencrypt, see the note below.
+
+* `TRAEFIK_ACME_CASERVER` +
+Set the CAServer to staging, see the note below.
+
+* `OCIS_DOMAIN`, `COLLABORA_DOMAIN` and `WOPISERVER_DOMAIN` +
+Set the domain names as defined in xref:domain-names[Domain Names].
+
+* `SMPT_xxx` +
+Define these settings according your eMail configuration. With the settings defined, Infinite Scale is able to send notifications to users. If the settings are not defined, Infinite Scale will start, but notifications can't be sent.
+
+[NOTE]
+====
+* When not defining own domain names, internal evaluation only domain names with self signed certificates are used automatically.
+
+* To trigger cerfificate issuing via LetsEncrypt, it checks in the request for creating valid certificates if the response eMail address is valid and continues if so. The eMail address used is defined via the variable `TRAEFIK_ACME_MAIL`. Expect that self signed certificates are used if the traefik log contains per URL defined the message "Contact emails @example.org are forbidden".
+
+* We recommend *before using live certificates*, to use the https://letsencrypt.org/docs/staging-environment/[staging environment of Letsencrypt, window=_blank] which you can configure via `TRAEFIK_ACME_CASERVER`. If certificates can be created and are issued by `Fake LE intermediate X1`, you can switch back issuing valid certificates.
+====
+
+== Start the Compose Setup
+
+When you have finished the configuration, you can start the compose setup by issuing the following command:
+
+[source,bash]
+----
+sudo docker compose up --remove-orphans
+----
+
+This command will download all necessary containers and starts up the instance according your settings. All outputs and logging is also printed to the shell.
+
+For later startups, you can add the `-d` flag which starts the container in the background and no logging is printed to the shell.
+
+== First Time Login
+
+Now, after preparations have finished, you can access your instance *from any client*. To do so, open your browser and enter the instance URL:
+
+[source,URL]
+----
+ocis.yourdomain.com
+----
+
+Which will show the following screen:
+
+image::depl-examples/ubuntu-compose/infinite-scale-login.png[Infinite Scale Login, width=300]
+
+For the credentials, use:
+
+* `admin` as user and 
+* `admin` for the password, +
+or the one you have defined manually during setup.
+
+NOTE: If you forgot the manually defined password set during initial setup, you need to follow one of the procedures described in the xref:admin-password[Admin Password] section.
+
+If you have logged in successfully, you should see the following screen:
+
+image::depl-examples/ubuntu-compose/ocis-logged-in.png[Infinite Scale Logged In, width=300]
+
+Congratulations, you have successfully setup Infinite Scale based on the deployment example.
+
+TIP: Checkout the https://doc.owncloud.com/[Desktop App] or https://doc.owncloud.com/[Mobile Apps] the for enhanced usage.
```

Comment:
```suggestion
TIP: Checkout the https://doc.owncloud.com/[Desktop App] or https://doc.owncloud.com/[Mobile Apps]  for enhanced usage.
```

Reply:
This change is double present and already updated via the batch.